versioned_docs/version-2.0.0/concepts/installation.md

```diff
@@ -38,7 +38,7 @@ On MacOS and Windows, additional tools are required for Keploy due to the lack o
 Let's get started by setting up the Keploy alias with this command:
 
 ```bash
- curl -O https://raw.githubusercontent.com/keploy/keploy/main/keploy.sh && source keploy.sh
+ curl --silent -O -L https://keploy.io/install.sh && source install.sh
```

Comment:
<img width="1059" alt="image" src="https://github.com/keploy/docs/assets/53110238/3ab6d550-726c-4974-ac1e-7470d323a59c">

Reply:
Getting rm: keploy.sh issue after installation also the install.sh file does not get removed!

Reply:
https://github.com/keploy/keploy/pull/1878